src/llm/servable.cpp

```diff
@@ -87,8 +87,12 @@ absl::Status GenAiServable::prepareInputs(std::shared_ptr<GenAiServableExecution
         return absl::Status(absl::StatusCode::kInvalidArgument, "API handler is not initialized");
     }
 
-    std::string inputText;
+    // Base servable cannot process images
+    if (executionContext->apiHandler->getImageHistory().size() > 0) {
+        return absl::InternalError("This servable supports only text input, but image_url has been provided");
```

Comment:
how do we know that image_url was provided? we only see that no images has been sent